guides/advanced_guides/datatypes.md

```diff
@@ -0,0 +1,99 @@
+# Data Types
+
+This page describes the different data types supported for the fields in a document and how MeiliSearch does handle them.
+
+- [String](/guides/advanced_guides/datatypes.md#string)
+- [Numeric types](/guides/advanced_guides/datatypes.md#numeric-types): `integer`, `float`
+- [Boolean](/guides/advanced_guides/datatypes.md#boolean)
+- [Array](/guides/advanced_guides/datatypes.md#array)
+- [Object](/guides/advanced_guides/datatypes.md#object)
+- [`null` type](/guides/advanced_guides/datatypes.md#null-type)
+
+When documents are returned upon search, they contain [displayed fields](/guides/advanced_guides/field_properties.md#displayed-fields). The displayed values are always **returned unchanged**.
```

Comment:
Returned unchanged based on what, how they were added in MeiliSearch?
ex: 
Lets say I add the following document:
```
{
 "id": 1
 "a" : {
    "b: {
        c: {
           "d": "Hello"
       }     
    }
 }
}
```
Is the field `a` field going to be displayed exactly like this in the returned document?


Nevertheless I don't think this is a place to talk about searchable and displayed (see comment above for simplified suggestion)
```suggestion
```

Reply:
Agreed! Will rethink the whole guide.